ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/StateManager.java

```diff
@@ -511,6 +511,16 @@ public final Uid get_uid ()
         return objectUid;
     }
 
+    /**
+     * @return the time the object was created in milliseconds since midnight, January 1, 1970 UTC
+     * @see System.currentTimeMillis
+     */
```

Comment:
I'm going to approve the PR but I'm interested to know if your IDE added this blank line or yourself.
Note that at some point we are likely to update our checkstyle rules to disallow it.

Reply:
I followed the design of the file. Everywhere newlines between method and javadoc.

Reply:
Oh yes, you're correct, sorry about that.